src/api/github/appAuthStrategyOptions.ts

```diff
@@ -0,0 +1,7 @@
+// I didn't find something great in @octokit/types.
+
+export interface AppAuthStrategyOptions {
```

Comment:
should this belong in `src/types` instead?

Reply:
Oooh, good call! Done in b98db3a7bea4da859f66107157f227636c120bac.